ci/prow/config.yaml

```diff
@@ -115,10 +116,16 @@ presets:
   - name: account
     secret:
       secretName: release-account
+  - name: hub-token
```

Comment:
is this github token?

Reply:
Yes, but it's specifically to be used with hub because of the ACL.